src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -50,7 +50,15 @@ const DiscoverApps = React.memo(() => (
           </Icon>
           <Name>{app.name}</Name>
           <TagWrapper>
-            <Tag background={statuses[app.status]}>{app.status}</Tag>
+            <Tag
+              mode="new"
+              css={`
+                overflow: hidden;
+                text-overflow: ellipsis;
```

Comment:
Maybe we should provide these two by default?

Reply:
I hear you, and I've been thinking about it but I don't know, it seems the simplicity of Tag might be over engineered, then again, if we find that this is always the case it would make sense to provide these by default.